test/unit/processing/settlement/schema.test.js

```diff
@@ -0,0 +1,116 @@
+const schema = require('../../../../app/processing/settlement/schema')
+let settlement
+
+describe('settlement schema', () => {
+  beforeEach(() => {
+    settlement = {
```

Comment:
we have a `mock-settlement` file

Reply:
We do but it contains properties that aren't allowed here, eg `sourceSystem` and some missing that are `paymentRequestId`.